package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "my-calculator",
+  "version": "1.0.0",
+  "description": "A basic client side calculator",
+  "main": "calculator.html",
+  "scripts": {
+    "test": "node node_modules/mocha/bin/mocha spec --recursive"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/SMH110/My-Calculator.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/SMH110/My-Calculator/issues"
+  },
+  "homepage": "https://github.com/SMH110/My-Calculator#readme",
+  "devDependencies": {
```

Comment:
We have no runtime dependencies (which would have been a property called `dependencies` if we had any) because our code runs in the browser and gets knockout from a CDN. But we do have devDependencies. These are dependencies used for the development of our application code. We use mocha for our tests, chai for our assertions and we need knockout during testing.